test/gurka/test_gtfs.cc

```diff
@@ -0,0 +1,151 @@
+#include "gurka.h"
+#include "third_party/just_gtfs/include/just_gtfs/just_gtfs.h"
+
+#include "string.h"
+#include <gtest/gtest.h>
+
+using namespace gtfs;
+using namespace valhalla;
+
+#define tripOneID "bar"
+#define tripTwoID "barbar"
+#define stopOneID "foo"
+#define stopTwoID "foo2"
+#define shapeOneID "square"
+#define serviceOneID "bon"
```

Comment:
we shouldnt use macros for constants these should look like:

`const std::string tripOneID = "bar";`